supervisor/supervisor.go

```diff
@@ -291,6 +292,14 @@ func (s *Supervisor) Machine() Machine {
 
 // SendTask sends the provided event the the supervisors main event loop
 func (s *Supervisor) SendTask(evt Task) {
+	ctxVal := fmt.Sprintf("%v", evt.Ctx())
+	if evt.Ctx().Done() == nil && ctxVal == "context.Background" {
+		logrus.Debug("Background task found without Done Channel, close.")
```

Comment:
what I'm questioning here is why are we relying on Done (which is context internal) and context.Background.

Reply:
see prior comment.